app/controllers/hyrax/dashboard/profiles_controller_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+# OVERRIDE FILE from Hyrax v5.0.0
+
+##
+# Ensure the current user matches the requested URL.
+Hyrax::Dashboard::ProfilesController.before_action :users_match!, only: %i[show edit update]
```

Comment:
does this mean a user can only see their own profile and no one elses? I get edit/update, I question show